### ISSUE TITLE
Assignments

### ISSUE THREAD

**Issue by dojo-json** [CodingDojo-MEAN-05-18/RLee-codes#1]
Thanks for pushing your latest assignments.

**Comment by Unknonymous** [comment]
No problem!

Well, actually, one small problem...   I keep getting an error:   *"Unexpected
token < at position 0",* when I run my PokemonApi assignment.

As far as I can tell it's being caused by an html response when json is
expected, but I haven't been able to get it sorted.
Any chance you'd have a couple mins after class this evening to review it
with me?

-Ray

**Comment by dojo-json** [comment]
Certainly. That error is often encountered when sending back `html` when you expect `json`. I don't see any html in that assignment currently, but consider your `express.static` middleware intercepts `get` requests to `/` and automatically sends back an `index.html` if one is found. I would recommend making your API more RESTful by using the resource name, `pokemons`, at the beginning of each route for that content.

**Comment by Unknonymous** [comment]
Unfortunately, last night, I think I crashed out before pushing up my
latest changes...   They're there now though.

Even so, I think your suggestion will neatly solve my issue.
There's no doubt I'm hitting ("/") for my base route...
In the latest version, I have angular installed, and the skeleton of the
service setup.

I'll try to have your suggested changes in place before lecture.  (Will be
a bit of a time crunch since I won't be off work until just before class.)

Thanks!

Ray

**Comment by dojo-json** [comment]
I recommend closing this issue automatically with your next push. Check out the documentation for closing issues with keywords: [https://help.github.com/articles/closing-issues-using-keywords/ ](https://help.github.com/articles/closing-issues-using-keywords/ )

TLDR: Create a commit message that contains something like `closes #1`. Github will detect the keyword and close the issue.
